### PR TITLE
[Chore] Add public logs

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

To be able to reproduce the crash using TestFlight build, there are more public logs that can help us.

### Note
This should be reverted after debugging.

### CODE REVIEW

File: Source/ManagedObjectContext/CoreDataStack.swift [hunk 1]
```diff
@@ -308,7 +310,12 @@ public class CoreDataStack: NSObject, ContextProvider {
             context.mergePolicy = NSMergePolicy(merge: .mergeByPropertyObjectTrumpMergePolicyType)
 
             FeatureService(context: context).createDefaultConfigsIfNeeded()
-            context.saveOrRollback()
+
+            do {
+                try context.save()
+            } catch {
+                zmLog.safePublic("Failed to save context: \(error.localizedDescription)")
+            }
```
Comment (KaterinaWire) [inline Source/ManagedObjectContext/CoreDataStack.swift:318]: @johnxnguyen  It was a temporary solution to save the context here. We save it in the updateOrCreate () method. Why do you want to keep it?

Comment (johnxnguyen) [reply Source/ManagedObjectContext/CoreDataStack.swift:318]: ah, in that case we can remove it. As long as we have one place where we can log the context save.

Comment (johnxnguyen) [reply Source/ManagedObjectContext/CoreDataStack.swift]: Removed

**Review by johnxnguyen — APPROVED** [review]

**Review by typfel — APPROVED** [review]

**Comment by KaterinaWire** [comment]
release patch

**Pull request merged**